Build/Tactical/Soldier_Create.cc

```diff
@@ -1336,6 +1339,7 @@ void CreateDetailedPlacementGivenBasicPlacementInfo( SOLDIERCREATE_STRUCT *pp, B
 	pp->bWisdom				= (INT8)(bBaseAttribute + Random( 9 ) + Random( 8 ));
 	pp->bMorale				= (INT8)(bBaseAttribute + Random( 9 ) + Random( 8 ));
 
+	if(ubSoldierClass==SOLDIER_CLASS_ELITE && pp->bExpLevel>GCM->getGamePolicy()->enemy_elite_maximum_level)
```

Comment:
Looking at it, it's more correct to only run the next line if these two conditions are true.